rl4j/rl4j-core/src/main/java/org/deeplearning4j/rl4j/observation/transform/FilterOperation.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2019 Skymind, Inc.

```

Comment:
New classes (not moved from elsewhere, or mostly copy/paste from other pre-existing code) should be `Copyright (c) 2020 Konduit K.K.`
There's other instances, I won't flag all of them.
Also missing copyright headers on EncodableToINDArrayTransform for example

Reply:
You know, I'm not sure about the whole copyright thing. I think the copyright remains with the authors, and doesn't move to either Konduit or Eclipse from what I understand.

Reply:
See here: https://www.eclipse.org/legal/legalfaq.php#h.r2y1yck33rg4

Reply:
I have changed those that had a header. There are still a couple of files without headers, but if I fix them, that will bring this PR beyond 1000 lines. I can add the missing headers in the part 2 of this contribution, if you don't mind.